records_mover/airflow/hooks/records_hook.py

```diff
@@ -28,7 +28,9 @@ def _get_boto3_session(self) -> boto3.session.Session:
 
     @property
     def _url_resolver(self) -> UrlResolver:
-        return UrlResolver(boto3_session=self._get_boto3_session())
+        return UrlResolver(boto3_session_getter=self._get_boto3_session,
+                           gcs_client_getter=lambda: None,
+                           gcp_credentials_getter=lambda: None)
```

Comment:
I haven't converted this over to look up GCS GCP creds yet.  It seems like there's starting to be some overlap between RecordsHook and CredsViaAirflow.  

Could it be time to bring in the Session object here?  Or at least an instance of CredsViaAirflow?